### PR TITLE
fix: gh fetch error hidden

### PR DESCRIPTION
**Pull request by TomerFi**
@bennypowers 

fixed the exception suppression we discussed.

in regards to the CODEOWNERS file,
I'm of the fence, please share your take on this.

as it's designed so far, you and/or me are listed as owners for all the core stuff.
the _about_, _community_, and _posts_ folders, has no owners.

I can modify this of course, this is just a suggestion.
WDYT?

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+_data/ @TomerFi @bennypowers
+_includes/ @bennypowers
+assets/ @bennypowers
+feed/ @bennypowers
+/*.* @bennypowers @TomerFi
```
Comment (bennypowers) [inline .github/CODEOWNERS:5]: you'll get a ping for all the dot-files and such

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+.github/ @TomerFi
+*.njk @bennypowers @TomerFi
```
Comment (bennypowers) [inline .github/CODEOWNERS:7]: you'll get pinged on layout and templates stuff

File: _data/contributors.js [hunk 3]
```diff
@@ -68,12 +68,8 @@ module.exports = async function israelContributors(_configData) {
...
     console.log('  ...cache miss, fetching...');
-    try {
-      const result = await getStatsForOrgMembers(initialQuery);
-      cache.save(result, 'json');
-      return result;
-    } catch (error) {
-      console.error(error);
-    }
+    const result = await getStatsForOrgMembers(initialQuery);
+    cache.save(result, 'json');
```
Comment (bennypowers) [inline _data/contributors.js:72]: thanks for 'catch'ing this :drum:

**Review by bennypowers — APPROVED** [review]

**Comment by bennypowers** [comment]
LGTM if you're happy with the CODEOWNERS setup, resolve above notes to merge

File: .github/CODEOWNERS (hunk 1 above)
Comment (TomerFi) [reply .github/CODEOWNERS:5]: only for ones on the root (including root folders).
these are mainly core files no?

File: .github/CODEOWNERS (hunk 2 above)
Comment (TomerFi) [reply .github/CODEOWNERS:7]: yes, are they changing frequently?
I figured, once we get rolling, most changes would be in the posts folder no?

Comment (TomerFi) [reply .github/CODEOWNERS:7]: you know what... let's keep it like this for now.
I'll remove myself if it get out of hand.

**Pull request merged**